en/deploy-tidb-operator.md

```diff
@@ -39,39 +39,202 @@ Refer to [Local PV Configuration](configure-storage-class.md) to set up local pe
 
-2. Install TiDB Operator.
+    Other items such as `limits`, `requests` and `replicas`, please modify as needed.
+
+3. Install TiDB Operator.
 
     {{< copyable "shell-regular" >}}
 
     ```shell
-    helm install pingcap/tidb-operator --name=tidb-operator --namespace=tidb-admin --version=${chart_version} -f /home/tidb/tidb-operator/values-tidb-operator.yaml && \
+    helm install pingcap/tidb-operator --name=tidb-operator --namespace=tidb-admin --version=${chart_version} -f ${HOME}/tidb-operator/values-tidb-operator.yaml && \
     kubectl get po -n tidb-admin -l app.kubernetes.io/name=tidb-operator
     ```
 
+4. Upgrade TiDB Operator
+
+    If you need to upgrade the TiDB Operator, please modify the `${HOME}/tidb-operator/values-tidb-operator.yaml` file, and then execute the following command to upgrade:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm upgrade tidb-operator pingcap/tidb-operator -f  ${HOME}/tidb-operator/values-tidb-operator.yaml
+    ```
+
+#### TiDB Operator Offline Installation
+
+You need to install TiDB Operator offline by the following steps if your server cannot access the network:
+
+1. Download `tidb-operator` chart
+
+    If there is no external network on the server, you cannot configure the Helm repo to install TiDB Operator components and other applications. At this time, you need to download the chart file needed for cluster installation on a machine with an external network, and then copy it to the server.
+
+    Use the following command to download the `tidb-operator` chart file:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    wget http://charts.pingcap.org/tidb-operator-v1.1.0.tgz
+    ```
+
+    Copy the `tidb-operator-v1.1.0.tgz` file to the target server and extract it to the current directory:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tar zxvf tidb-operator.v1.1.0.tgz
+    ```
+
+2. Download Docker Images Used by TiDB Operator
+
+    If the server does not have an external network, you need to download and upload all Docker images used by TiDB Operator on the machine with an external network to the server, and then use `docker load` to install the Docker image on the server.
+
+    The Docker images used by TiDB Operator are:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    pingcap/tidb-operator:v1.1.0
+    pingcap/tidb-backup-manager:v1.1.0
+    bitnami/kubectl:latest
+    pingcap/advanced-statefulset:v0.3.3
+    k8s.gcr.io/kube-scheduler:v1.16.9
+    ```
+
+    Among them, `k8s.gcr.io/kube-scheduler:v1.16.9` should be consistent with the version of your Kubernetes cluster, you don’t need to download it separately.
+
+    Next download all these images with the following command:
```

Comment:
```suggestion
    Next, download all these images using the following command:
```
`with` can have multiple meanings. Use `using` to avoid ambiguity.